shared/actions/platform-specific/index.desktop.js

```diff
@@ -245,7 +245,7 @@ const onConnected = () => {
       argv: process.argv,
       clientType: RPCTypes.commonClientType.guiMain,
       desc: 'Main Renderer',
-      pid: process.pid,
+      pid: SafeElectron.getRemote().process.pid,
```

Comment:
@keybase/react-hackers is this change ok? before the pid was the renderer's pid, now it is the main process's pid so killing it kills everything